libraries/ms-common/src/stm32f0xx/wait.c

```diff
@@ -0,0 +1,7 @@
+#include "wait.h"
+
+#include "stm32f0xx.h"
+
+void wait(void) {
+  __WFE();
```

Comment:
Done. I read about the debugging functions of `__WFE()` which is why I went with that but on second thought you are correct.